packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -96,6 +96,11 @@ class PostPublishPanel extends Component {
 							{ PostPublishExtension && <PostPublishExtension /> }
 						</PostPublishPanelPostpublish>
 					) }
+					<CheckboxControl
+						label={ __( "Don't show this sidebar again." ) }
```

Comment:
This should be an apostrophe (`’`, not a single-quote `'`).

Reply:
I need to update my [keyboard setup](http://nosolosoftware.com/touch-typing-in-dvorak/) to include that apostrophe. Good catch!

Reply:
874a40b064cec0e818b8e7f9b31fd592c039bb66